src/NuGet.Clients/NuGet.PackageManagement.UI/Resources.resx

```diff
@@ -432,6 +435,9 @@ Packages installed into Website projects will not be restored during build. Cons
   <data name="Text_ViewLicense" xml:space="preserve">
     <value>View License</value>
   </data>
+  <data name="Text_ViewReadme" xml:space="preserve">
+    <value>View README</value>
```

Comment:
Seems to me it'd be better to keep this a static wording. 
Our other links show the full URL.... I'm not sure how great that would be, but we are doing it in 2 other links, there.
Have we/should we run this by design team?

Reply:
> it also has the issue of users often thinking that it'll open locally rather than open in web.

> Our other links show the full URL.... I'm not sure how great that would be, but we are doing it in 2 other links, there.

These are good points that makes me lean towards showing the full URL. Do we know what the URL would be?  Something like `https://www.nuget.org/packages/newtonsoft.json/12.0.3/readme` would seem pretty clean to me.

Reply:
It'll look somewhat similar to that. We tested it with this URL: `https://dev.nugettest.org/packages/embeddedreadmefiletest2/3.0.0/#show-readme-container` (will end up being `www.nuget.org` instead of `dev.nuggettest.org`) but I imagine this can be cleaned up further.